app/api/tickets.py

```diff
@@ -36,6 +37,9 @@ def before_post(self, args, kwargs, data):
         if get_count(db.session.query(Ticket.id).filter_by(name=data['name'], event_id=int(data['event']))) > 0:
             raise ConflictException({'pointer': '/data/attributes/name'}, "Ticket already exists")
 
+        if get_count(db.session.query(Event).filter_by(id=int(data['event']), is_ticketing_enabled=False)) > 0:
+            raise ConflictException({'pointer': ''}, "Ticketing is disabled for this Event")
```

Comment:
Shouldn't be ConflictException. Should be not allowed HTTP status code

Reply:
Okay, I've made the changes. Please see if it's fine.